backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/MeetingService.cs

```diff
@@ -187,15 +210,23 @@ public async Task SendEmailsAsync(long meetingId, TemplateType type)
                 .Concat(meeting.ExternalAttendees.Select(attendee => attendee.Email))
                 .ToList();
 
-            if (emailTemplate is null || recipients is null || !recipients.Any())
+            if (recipients is null || !recipients.Any() || meeting is null)
             {
                 return;
             }
 
-            var emailDto = _mapper.Map<EmailDto>(emailTemplate);
+            if (emailTemplate is not null)
+            {
+                emailDto = _mapper.Map<EmailDto>(emailTemplate);
+            }
 
             foreach (var recipient in recipients)
             {
+                if (emailTemplate is null)
+                {
+                    emailDto = GenerateNewMeetingEmailTemplate(meeting, recipient);
+                }
```

Comment:
Why do we need two these checks?

Reply:
We use this method for new meetings with "+ New Meeting" and Availability Slot. If there are any template, it`s "+ New Meeting" and method must generate template.